README.md

```diff
@@ -24,12 +24,36 @@ Or if you want to use the latest Feathr code from GitHub:
 pip install git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project
 ```
 
-## Quick Start
+## Running Feathr with only a few clicks
 
-- Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb) to try it out. There is also a companion [quick start guide](./docs/quickstart.md) containing a bit more explanation on the notebook.
-- For more details, read our [documentation](https://linkedin.github.io/feathr/).
+Feathr has native cloud integration and getting started with Feathr is very straightforward. You only need three steps:
 
-## Defining Features with Transformation
+1. Get the principal ID of your account by running `az ad signed-in-user show --query objectId -o tsv` in the link below (Select "Bash" if you are asked to choose one), and write down that value (will be something like `b65ef2e0-42b8-44a7-9b55-abbccddeefff`)
+
+
```

Comment:
we should setup expectations for this.

What's this? Is it just a minimal example for users to test or product ready? Let's describe that.

Reply:
Good point